Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -44,6 +43,23 @@ public IEnumerable ItemsSource
 			set => SetValue(ItemsSourceProperty, value);
 		}
 
+		public static readonly BindableProperty RemainingItemsThresholdReachedCommandProperty = BindableProperty.Create(nameof(RemainingItemsThresholdReachedCommand), typeof(ICommand), typeof(ItemsView), null);
+
+		public ICommand RemainingItemsThresholdReachedCommand
+		{
+			get { return (ICommand)GetValue(RemainingItemsThresholdReachedCommandProperty); }
+			set { SetValue(RemainingItemsThresholdReachedCommandProperty, value); }
+		}
+
+		public static readonly BindableProperty RemainingItemsThresholdProperty =
```

Comment:
Should these properties be added to `CollectionView` directly instead of the base class? I guess there is a possibility that somebody might want to lazy load `CarouselView` items.

Reply:
Yes, these should be on ItemsView so "infinite" CarouselViews are possible.